<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStatusBar {
color: #FF0000;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>翻译导入导出</string>
      </property>
      <widget class="QPushButton" name="tsImportBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ts导入</string>
       </property>
      </widget>
      <widget class="QPushButton" name="generateBtn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>excel生成</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tsUpdateBtn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>译文写入</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>30</y>
         <width>321</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLineEdit" name="tsPathEdit">
          <property name="text">
           <string>C:/Users/xianl/Documents/string.ts</string>
          </property>
          <property name="placeholderText">
           <string>ts文件路径</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="tsLookBtn">
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="excelPathEdit">
          <property name="text">
           <string>C:/Users/xianl/Documents/export.xlsx</string>
          </property>
          <property name="placeholderText">
           <string>excel文件路径</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="excelLookBtn">
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>翻译所在列</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="transSpinBox">
          <property name="minimum">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>有道翻译</string>
      </property>
      <widget class="QPushButton" name="translateBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>有道翻译</string>
       </property>
      </widget>
      <widget class="QLabel" name="youdaoDstLabel_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>331</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>注意：会翻译到上面指定的Excel所在列</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>281</width>
         <height>115</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="youdaoAppIdlineEdit">
          <property name="text">
           <string>3024f8dc8e60f56d</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="youdaoDstLabel">
          <property name="text">
           <string>目标语言</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="youdaoAppIdLabel">
          <property name="text">
           <string>应用ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="youdaoKeylineEdit">
          <property name="text">
           <string>RYwu8rvyeBqTL8NGnMUVWWUdB42qgASp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="youdaoKeyLabel">
          <property name="text">
           <string>应用密钥</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
